analytics_data_api/v0/views/learners.py

```diff
@@ -251,33 +267,33 @@ class CourseLearnerMetadata(CourseViewMixin, generics.RetrieveAPIView):
+            * cohorts: An object that maps the names of cohorts in the course
+              to the number of learners belonging to those cohorts.
+            * segments: An object that maps the names of segments in the course
+              to the number of learners belonging to those segments. The
+              current set of segments is "highly_engaged", "disengaging",
               "struggling", "inactive", and "unenrolled".
-            * enrollment_modes: An object mapping the names of enrollment modes
-              in the course to the number of students belonging to those
-              enrollment modes.  Examples include "honor" and "verified".
+            * enrollment_modes: An object that maps the names of enrollment
+              modes in the course to the number of learners belonging to those
+              enrollment modes. Examples include "audit" and "verified".
             * engagement_ranges: An object containing ranges of learner
-              engagement with the courseware.  Each range has 'below_average',
-              'average', and 'above_average' keys which map to two-element
-              arrays of which the first element is the lower bound (inclusive)
-              and the second element is the upper bound (exclusive).  It has
-              the following keys:
-                * date_range: The time duration for which this data applies
-                * problems_attempted: engagement ranges for the number of
-                  problems attempted in the date range.
-                * problems_completed: engagement ranges for the number of
-                  problems completed in the date range.
-                * problem_attempts_per_completed: engagement ranges for the
+              engagement with the courseware. Each range has 'below_average',
+              'average', and 'above_average' keys. These keys map to
+              two-element arrays, in which the first element is the lower bound
+              (inclusive) and the second element is the upper bound
+              (exclusive). It has the following keys.
+                * date_range: The time period to which this data applies.
+                * problems_attempted: Engagement ranges for the number of
+                  unique problems attempted in the date range.
+                * problems_completed: Engagement ranges for the number of
+                  unique problems answered correctly in the date range.
+                * problem_attempts_per_completed: Engagement ranges for the
                   number of problem attempts per completed problem in the date
                   range.
-                * discussions_contributed: engagement ranges for the number of
-                  discussions contributed in the date range.
+                * discussions_contributed: Engagement ranges for the number of
+                  times learners participated in discussions in the date range.
```

Comment:
The way @lamagnifica has worded it here is correct. We are not counting unique contributions, we are counting *any* posts, comments, or responses they create.